docs/publish/centralized-deployment.md

```diff
@@ -126,20 +126,21 @@ If the add-in supports add-in commands, the commands will appear on the Office a
 
 If the add-in does not support add-in commands, users can add it to their Office application by doing the following:
 
-1.	In Word 2016, Excel 2016, or PowerPoint 2016, choose **Insert** > **My Add-ins**.
+1.	In Word, Excel, or PowerPoint, choose **Insert** > **My Add-ins**.
```

Comment:
Also, do same instructions apply across the various versions of Office?

Reply:
Fixed.